sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAsyncClient.java

```diff
@@ -0,0 +1,513 @@
+import com.azure.communication.callingserver.implementation.converters.InviteParticipantsConverter;
+import com.azure.communication.callingserver.implementation.converters.PlayAudioConverter;
+import com.azure.communication.callingserver.implementation.models.PhoneNumberIdentifierModel;
+import com.azure.communication.callingserver.implementation.models.PlayAudioRequest;
+import com.azure.communication.callingserver.models.CallModality;
+import com.azure.communication.callingserver.implementation.models.CancelAllMediaOperationsRequest;
+import com.azure.communication.callingserver.implementation.models.CreateCallRequestInternal;
+import com.azure.communication.callingserver.models.EventSubscriptionType;
+import com.azure.communication.callingserver.implementation.models.InviteParticipantsRequest;
+import com.azure.communication.callingserver.models.CancelAllMediaOperationsResponse;
+import com.azure.communication.callingserver.models.CreateCallOptions;
+import com.azure.communication.callingserver.models.CreateCallResponse;
+import com.azure.communication.callingserver.models.PlayAudioOptions;
+import com.azure.communication.callingserver.models.PlayAudioResponse;
+import com.azure.communication.common.CommunicationIdentifier;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Async client that supports server call operations.
+ */
+@ServiceClient(builder = CallClientBuilder.class, isAsync = true)
+public final class CallAsyncClient {
+    private final CallsImpl callClient;
+    private final ClientLogger logger = new ClientLogger(CallAsyncClient.class);
+
+    CallAsyncClient(AzureCommunicationCallingServerServiceImpl callServiceClient) {
+        callClient = callServiceClient.getCalls();
+    }
+
+    /**
+     * Create a Call Request from source identity to targets identity.
+     *
+     * @param source The source of the call.
+     * @param targets The targets of the call.
+     * @param createCallOptions The call Options.
+     * @return response for a successful CreateCall request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<CreateCallResponse> createCall(CommunicationIdentifier source,
+                                               CommunicationIdentifier[] targets,
+                                               CreateCallOptions createCallOptions) {
+        try {
+            Objects.requireNonNull(source, "'source' cannot be null.");
+            Objects.requireNonNull(targets, "'targets' cannot be null.");
+            Objects.requireNonNull(createCallOptions, "'CreateCallOptions' cannot be null.");
```

Comment:
nit: `createCallOptions` lower first letter, this should be the variable name

Reply:
fixed